pkgs/development/python-modules/beatprints/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  fetchFromGitHub,
+  pillow,
+  buildPythonPackage,
+  poetry-core,
+  requests,
+  pylette,
+  lrclibapi,
+  fonttools,
+  questionary,
+  rich,
+  toml,
+}:
+buildPythonPackage rec {
```

Comment:
Since this is an application it should go in pkgs/by-name and use buildPythonApplication.